source/framework/core/src/TRestAnalysisTree.cxx

```diff
@@ -591,11 +591,11 @@ void TRestAnalysisTree::PrintObservables() {
     }
 
     cout.precision(15);
-    std::cout << "Entry : " << fReadEntry << std::endl;
-    std::cout << "> Event ID : " << GetEventID() << std::endl;
-    std::cout << "> Event Time : " << GetTimeStamp() << std::endl;
-    std::cout << "> Event Tag : " << GetSubEventTag() << std::endl;
-    std::cout << "-----------------------------------------" << std::endl;
```

Comment:
Do you understand why we have to revert `std::endl` to `endl`

Reply:
> Do you understand why we have to revert `std::endl` to `endl`

Can you rephrase?

Reply:
Does it arises from `struct endl_t` on `TRestStringOutput.h`? I think we should fix it in this PR...

Reply:
I am not sure if I would complicate more this PR as its already big enough, perhaps @jgalan has a opinion. I agree that this should be fix, but perhaps on another PR if its not an easy fix. So far this doesn't seem to cause any problem, but it may have been unnoticed.

Reply:
`TRestStringOutput.h` was designed by @nkx, probably he has a better opinion than myself.

On the other hand, you need to create a new branch `lobis-remove-namespace-std-from-headers` at rawlib to fix compilation issues at this PR.

Reply:
> On the other hand, you need to create a new branch `lobis-remove-namespace-std-from-headers` at rawlib to fix compilation issues at this PR.

But this branch already exists, I don't understand the error. https://github.com/rest-for-physics/rawlib/tree/lobis-remove-namespace-std-from-headers

Reply:
> The problems with endl arise from TRestStringOutput.h

What is the problem? In principle it is OK to remove `using namespace std`, because once we do that, we need to state `std::end`, which induces less ambiguity to the compiler. There will be less problems for my `endl_t`

Reply:
> > The problems with endl arise from TRestStringOutput.h
> 
> What is the problem? In principle it is OK to remove `using namespace std`, because once we do that, we need to state `std::end`, which induces less ambiguity to the compiler. There will be less problems for my `endl_t`

If I remember correctly one of the problems is that you need to use `std::endl` (explicitly, event when `using namespace std` is declared) when using `std::cout` or it will throw an error. Also you need to use `endl` (without `std::`) when using `metadata <<` and such or it will throw an error. I think it shouldn't behave like this.